docs/_markbind/layouts/default.md

```diff
@@ -31,7 +31,7 @@
   * [Quick Start]({{ baseUrl }}/UserGuide.html#quick-start)
   * [Features]({{ baseUrl }}/UserGuide.html#features)
   * [FAQ]({{ baseUrl }}/UserGuide.html#faq)
-  * [Command Summary]({{ baseUrl }}/UserGuide.html#faq)
+  * [Command Summary]({{ baseUrl }}/UserGuide.html#command-summary)
```

Comment:
Can move up also? A bit weird if command summary is on the very top but navbar suggests otherwise

Reply:
Thanks for pointing that out. I have made the changes.